pubspec.yaml

```diff
@@ -22,7 +22,8 @@ dependencies:
   stream_channel: ^2.1.2
   ffi: ^2.0.2
   big_decimal: ^0.5.0
+  path_provider: ^2.1.3
```

Comment:
I'm getting this error when updating the packages, since I'm not using flutter:
```
[znn_sdk_dart] dart pub upgrade
Resolving dependencies...
Because znn_sdk_dart depends on path_provider >=0.4.0 which requires the Flutter SDK, version solving failed.

Flutter users should use `flutter pub` instead of `dart pub`.
exit code 1
```

I think it would be good if this SDK can be used without needing the Flutter SDK since many use cases don't need it.